packages/shared/src/components/ExtensionOnboarding.tsx

```diff
@@ -18,6 +18,13 @@ const ExtensionOnboarding = (): ReactElement => {
     };
   }, [setCurrentPage]);
 
+  const onboarding = (() => {
+    const base = new URL(onboardingUrl);
```

Comment:
just a note f this component is used in webapp where `webappUrl` is just `/`, this needs a base URL new URL(onboardingUrl, base). But not blocking